src/server/src/v1/v1.module.ts

```diff
@@ -1,14 +1,13 @@
 import { Module } from '@nestjs/common'
-import { HealthController } from './health.controller'
-import { ActivitiesService } from './activities.service'
-import { PulseRepo } from '../db/pulse.repo'
-import { PulseController } from './pulse.controller'
-import { WakatimeController } from './wakatimeProxy.controller'
-import { StartupService } from './startup.service'
+import { ActivitiesService } from './pulse/application/services/activities.service'
+import { PulseRepo } from './pulse/infrastructure/database/pulse.repo'
+import { PulseController } from './pulse/infrastructure/http/controllers/pulse.controller'
+import { WakatimeController } from './pulse/infrastructure/http/controllers/wakatimeProxy.controller'
+import { StartupService } from './startup/application/services/startup.service'
 
 @Module({
   imports: [],
-  controllers: [HealthController, PulseController, WakatimeController],
```

Comment:
Does nest come with a health check?

Reply:
I forgot to add it to the common folder, now is in the right way